extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeCommentsInfoItemExtractor.java

```diff
@@ -98,4 +99,18 @@ public String getUploaderUrl() throws ParsingException {
         return ServiceList.PeerTube.getChannelLHFactory()
                 .fromId("accounts/" + name + "@" + host, baseUrl).getUrl();
     }
+
+    @Override
+    public Page getReplies() throws ParsingException {
+        if (JsonUtils.getNumber(item, "totalReplies").intValue() == 0) {
+            return null;
```

Comment:
Should the method be annotated as `@Nonnull`?

Reply:
Did you mean `@Nullable`?